deployment/ccip/changeset/cs_active_candidate_test.go

```diff
@@ -98,7 +98,7 @@ func TestActiveCandidate(t *testing.T) {
 	_, err = commonchangeset.ApplyChangesets(t, e, timelocks, []commonchangeset.ChangesetApplication{
 		// note this doesn't have proposals.
 		{
-			Changeset: commonchangeset.WrapChangeSet(commonchangeset.NewTransferOwnershipChangeset),
+			Changeset: commonchangeset.WrapChangeSet(commonchangeset.TransferToMCMSWithTimelock),
 			Config:    genTestTransferOwnershipConfig(tenv, allChains, state),
 		},
```

Comment:
Can't comment on the exact line, but I believe you can axe the NewAcceptOwnershipChangeset? Or do we wanna keep it around for other reasons?

Reply:
Can at least axe the next ChangesetApplication from this particular input slice.

Reply:
yeah removing